.github/workflows/lifecycle-image.yml

```diff
@@ -0,0 +1,30 @@
+name: lifecycle-image
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get metadata
+        id: get_metadata
+        run: |
+          - echo ::set-output name=LIFECYCLE_VERSION::`echo ${{ github.event.release.tag_name }} | cut -d "v" -f2`
+          - echo ::set-output name=BUILDPACK_API::`cat Makefile | grep BUILDPACK_API?= | cut -d "=" -f2`
+          - echo ::set-output name=PLATFORM_API::`cat Makefile | grep PLATFORM_API?= | cut -d "=" -f2`
```

Comment:
We should really make this data easier to get, but there aren't many better options right now. The alternative would be to extract the descriptor from the archive, but I am okay with this so long as we recognize it's a bit brittle and should be improved (probably with the help of better formatted lifecycle releases) when we can get around to it.